### ISSUE TITLE
Test running playbooks inside a container

### ISSUE THREAD

**Issue by miquecg** [miquecg/elixir-ide#59]
See: https://developers.redhat.com/blog/2019/04/04/build-and-run-buildah-inside-a-podman-container/

#### Sample usage
```
podman pull docker://quay.io/buildah/stable:latest

podman run stable buildah version

# Because both the container and the container within the container 
# will be using fuse-overlayfs, they won’t be happy trying to mount
# their respective directories over each other. First step is to create
# a directory for the container within the container to use.
mkdir /var/lib/mycontainer

# Run the image detached using the host's network in a container name
# buildahctr, turn off label and seccomp confinement in the container
# and then do a little shell hackery to keep the container up and running.
podman run --detach --name=buildahctr --net=host --security-opt label=disable --security-opt seccomp=unconfined --device /dev/fuse:rw -v /var/lib/mycontainer:/var/lib/containers:Z stable sh -c 'while true; do sleep 100000; done'

podman exec -it buildahctr /bin/sh

# Now inside of the container

buildah from alpine

buildah images

exit
```

#### Rootless Buildah
https://opensource.com/article/19/3/tips-tricks-rootless-buildah

**Comment by miquecg** [comment]
I've managed to run `build.yml` playbook inside a Podman container, although I needed to make it privileged because `arch_container` role requires to mount the container and some devices.

This poses a problem on CircleCI to directly use a container as executor, because they are unprivileged by default. Another option would be to continue using machine executors and pull a configured image to run build steps on it.
